vlab-research/fly
# Feature/websurvey logic jumps

### What changes have you made?
- referenced the [replybot](https://github.com/vlab-research/replybot) repository to refactor the existing web survey logic
- updated the file structure so that it's aligned with `replybot`
- installed dependencies and set up some basic unit tests using `mocha` and `chai` 
- adapted the unit tests in `replybot` to test the business logic in `web-survey`
- implemented the new unit tested functions at component level 
- set up logic jump functionality

### Which issue(s) does this PR address?
#52

### How to test
- use the terminal to see that all tests are passing
- ensure that functionality remains the same i.e. you are able to navigate linearly through the form and a thankyou screen displays at the end except for those questions with logic in which case the form should jump to the appropriate question (_this should only apply to question 1 so far_)
